apix-impl/src/main/java/eu/xenit/apix/alfresco/dictionary/AspectService.java

```diff
@@ -46,6 +47,10 @@ private AspectDefinition GetAspectDefinition(org.alfresco.service.namespace.QNam
             properties.add(c.apix((qName)));
         }
         ret.setProperties(properties);
+        List<QName> mandatoryAspects = aspectDef.getDefaultAspects(true).stream()
```

Comment:
This is including inherited aspects.
Is that on purpose?

I'm asking because there is also `getDefaultAspects()` (without boolean parameter) which also includes the inherited aspects.

Reply:
I don't have a strong opinion, but isn't it better to have a complete overview of aspects?

Reply:
Yes this was on purpose. 
The only argument pro local-aspects-only (local apects) I see is that when specifying a type, there is less elements to iterate/check.
But when creating a new node, or changing to a type that does not inherit from the current, it would be easy to lose/forget inherited aspects.
With that in mind, I suggest we go for completeness and pass the inherited aspects too.

as for the used method, we can indeed use the argumentless version. ill fix that.

Reply:
So checking this again, and the `getDefaultAspects()` returns intherited apects by default, so I'd count that as an additional argument to continue that here